docs/_links.rst

```diff
@@ -0,0 +1,33 @@
+.. _Apiary: https://apiary.io
+.. _Dredd: https://dredd.org
+.. _Gavel: https://github.com/apiaryio/gavel.js
+
+.. API Blueprint
+
+.. _API Blueprint: https://apiblueprint.org
+.. _MSON: https://apiblueprint.org/documentation/mson/tutorial.html
+
+
+.. OpenAPI specification
+
+.. _OpenAPI 2: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md
+.. _OpenAPI 3: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md
+
+
+.. JSON Schema
+
+.. _Draft 3: https://tools.ietf.org/html/draft-zyp-json-schema-03
```

Comment:
Down to you, but I feel like these references are a little ambiguous `Draft 3` could mean anything. I think it would be better to name them with `JSON Schema` in the name.